### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by sacha-l**
This PR fixes some issues tagged with the "broken-link" label. See commit history.

### CODE REVIEW

File: .netlify/_redirects_production [hunk 1]
```diff
@@ -123,6 +125,7 @@
 /docs/en/knowledgebase/advanced/no-hash-collections   /v3/advanced/hash-collections     301!
 /docs/en/knowledgebase/advanced/no-hash-collections/  /v3/advanced/hash-collections     301!
 /docs/en/knowledgebase/advanced/*             /v3/advanced/:splat                       301!
+/docs/low-level-data-formats                          /v3/advanced/scale-codec/         301!
```
Comment (NukeManDan) [inline .netlify/_redirects_production:128]: can we add one more for #492 ?

File: src/components/DevNavMenu.tsx [hunk 2]
```diff
@@ -540,24 +540,7 @@ DevNavMenuTuts.set('firstChain', [
...
-      {
-        title: `Build a Custom Pallet`,
-        link: '#building-a-custom-pallet',
-      },
-      {
-        title: `Build a Custom Front-end`,
-        link: '#building-a-custom-front-end',
-      },
-    ],
+    items: [ ],
```
Comment (NukeManDan) [inline src/components/DevNavMenu.tsx:543]: does this render as a dropdown still?

**Review by NukeManDan — APPROVED** [review]
lgtm just a request and question

File: src/components/DevNavMenu.tsx (hunk 2 above)
Comment (sacha-l) [reply src/components/DevNavMenu.tsx:543]: The modal does but it doesn't do anything. Hack for now until we figure out new UX and restructuring going forward. 🤷🏻

**Pull request merged**